extensions/windowgram/init.lua

```diff
@@ -0,0 +1,178 @@
+--- === hs.windowgram ===
+---
+--- auto-layout using ascii rows similar to tmuxomatic
+
+local windowgram = {}
+
+local alert = require 'hs.alert'
```

Comment:
this can go (wasn't using it except in commented code, that you rightfully removed)